packages/graph-engine/package.json

```diff
@@ -36,7 +36,7 @@
   },
   "dependencies": {
     "@bundled-es-modules/deepmerge": "^4.3.1",
-    "@tokens-studio/types": "^0.3.0",
+    "@tokens-studio/types": "^0.5.1",
```

Comment:
it should honestly probably be a regular dep. Peer deps are meant for things like plugins e.g. a prettier plugin has a peer dep on prettier using a version range for which the plugin is compatible. 

So for example, it makes sense for our nodes-audio workspace to have graph engine as a peer dep, since it's a plugin for it.

Reply:
updated